common/lib/permissions.js

```diff
@@ -87,6 +87,8 @@ const ocaPermissions = [
   'move:view',
   'move:create',
   'move:create:prison_transfer',
+  'move:create:secure_childrens_home',
```

Comment:
Can we get some E2E tests for these additions, so that we're protected in the future if requirements change?

Reply:
Added issue https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/issues/827